project/BuildHelper.scala

```diff
@@ -201,28 +184,22 @@ object BuildHelper {
 
   def stdSettings(prjName: String) = Seq(
     name                                   := s"$prjName",
-    crossScalaVersions                     := Seq(Scala211, Scala212, Scala213, Scala3),
+    crossScalaVersions                     := Seq(Scala212, Scala213, Scala3),
     ThisBuild / scalaVersion               := Scala213,
     scalacOptions ++= stdOptions ++ extraOptions(scalaVersion.value, optimize = !isSnapshot.value),
     libraryDependencies ++= {
-      if (scalaVersion.value == Scala3)
-        Seq(
-          "com.github.ghik" % s"silencer-lib_$Scala213" % SilencerVersion % Provided
```

Comment:
I removed Silencer because, from its doc:
> NOTE: Scala 2.13.2 and 2.12.13 introduced https://github.com/scala/scala/pull/8373. This means that unless you're still cross compiling for Scala 2.11, this plugin is obsolete, and you should use [@nowarn](https://www.scala-lang.org/api/current/scala/annotation/nowarn.html).

See https://github.com/ghik/silencer